assets/src/edit-story/components/panels/textStyle/useRichTextFormatting.js

```diff
@@ -133,6 +134,26 @@ function useRichTextFormatting(selectedElements, pushUpdate) {
     };
   }, [hasCurrentEditor, selectionActions, push]);
 
+  useGlobalKeyDownEffect(
+    { key: ['mod+b', 'mod+u', 'mod+i'] },
+    ({ key }) => {
+      switch (key) {
+        case 'b':
+          handlers.handleClickBold();
```

Comment:
This does both: whole text and selection bolding?

Reply:
Yes! If the box is in "edit mode" it just bolds the selection, where if you have just the box selected it bolds the entire thing. Mirrors the functionality of the button in the panel.